src/codemodder/codemods/utils_mixin.py

```diff
@@ -17,24 +17,41 @@
 class NameResolutionMixin(MetadataDependent):
     METADATA_DEPENDENCIES: Tuple[Any, ...] = (ScopeProvider,)
 
+    def find_imported_name(self, node: cst.Name):
```

Comment:
I would make this "private". The original idea was to make `find_base_name` a single stop for everything.